index.html

```diff
@@ -0,0 +1,262 @@
+                  <a href="#" class="btn btn-primary">Read More</a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md">
+              <div class="card bg-secondary text-light">
+                <div class="card-body text-center">
+                  <div class="h1 mb-3">
+                    <i class="bi bi-person-square"></i>
+                  </div>
+                  <h3 class="card-title mb-3">LH Exibition</h3>
+                  <p class="card-text">
+                    Appreciation of various creations applying 
+                    LH technological models organized from Kanya...
+                  </p>
+                  <a href="#" class="btn btn-primary">Read More</a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md">
+              <div class="card bg-secondary text-light">
+                <div class="card-body text-center">
+                  <div class="h1 mb-3">
+                    <i class="bi bi-person-square"></i>
+                  </div>
+                  <h3 class="card-title mb-3">Forums</h3>
+                  <p class="card-text">
+                    Have the time to share your thoughts and opinions with exparts from various disciplines...
+                  </p>
+                  <a href="#" class="btn btn-primary">Read More</a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md">
+              <div class="card bg-secondary text-light">
+                <div class="card-body text-center">
+                  <div class="h1 mb-3">
+                    <i class="fa fa-building" aria-hidden="true"></i>
+                    
+                  </div>
+                  <h3 class="card-title mb-3">Workshop</h3>
+                  <p class="card-text">
+                    Participants will create their mockups of the innovations instead of watching and just having theories...
+                  </p>
+                  <a href="#" class="btn btn-primary">Read More</a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md">
+              <div class="card bg-secondary text-light">
+                <div class="card-body text-center">
+                  <div class="h1 mb-3">
+                    <i class="bi bi-people"></i>
+                  </div>
+                  <h3 class="card-title mb-3">LH Ignite</h3>
+                  <p class="card-text">
+                    Get opportunity to network with LH affiliates around the world also after the summit. Thanks to LH ...
+                  </p>
+                  <a href="#" class="btn btn-primary">Read More</a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="mt-5">
+            <a href="#" class="see-program text-white mt-5"> SEE THE WHOLE PROGRAM</a>
+          </div>
+        </div>
+      </div>
+
+    </section>
```

Comment:
- [ ] According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Colors, Typography, and Layout.
The main programs section doesn't look similar to the original design, so kindly use the same font-colors and remove the `read more` button, and add the horizontal line below the title 👌💯

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181265557-79d56bf3-be93-4867-a941-a9401f6a4a3e.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181265627-fe7ea2d0-26b7-4475-b26e-ff369ebdde83.PNG" width="500" /> |

- [ ] Also, on the mobile version, you should update/change the style of the `SEE THE WHOLE PROGRAM` Link 👌

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181266211-4f6acb5a-9327-4fde-94ad-7d2e5d71ff2f.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181266102-bcbcac63-9707-44de-b805-4acf0219c319.PNG" width="500" /> |